vkconfig_core/layers/layers_schema.json

```diff
@@ -126,6 +60,11 @@
         }
     },
     "definitions": {
+        "version": {
```

Comment:
Possible rename to api_version? since its used for that purpose.

Reply:
I am using version as I use it for both api_version and file_format_version